lib/assertions.js

```diff
@@ -72,6 +73,23 @@ export default class DOMAssertions {
     notFocused.call(this, message);
   }
 
+  /**
+   * Assert that the [HTMLElement][] or an [HTMLElement][] matching the
+   * `selector` contains the provided `property` and optionally checks
+   * if the property `value` matches the provided `value`.
+   *
+   * @param {string} property
+   * @param {object?} options
```

Comment:
I'm wondering if we should mimic the `hasAttribute` API instead which accepts `name, value?, message?` 🤔

Reply:
+1 to this. There's a lot to be said for the API being consistent for like methods/functions.